src/components/profile/ViewProfileForm.jsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable camelcase */
+import { connect } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import 'react-toastify/dist/ReactToastify.css';
+
```

Comment:
I don't see the reason for importing ReactToastisfy.css because it is already imported in our main component App.jsx